src/app/tasks/task-comments-viewer/task-comments-viewer.component.ts

```diff
@@ -1,39 +1,49 @@
-import { Component, OnInit, Input, Inject, OnChanges, SimpleChanges, Output, EventEmitter } from '@angular/core';
-import { TaskCommentService } from 'src/app/common/services/task-comment.service';
+import { Component, OnInit, Input, Inject, OnChanges, SimpleChanges, ViewChild, ElementRef } from '@angular/core';
+import { TaskCommentViewService } from 'src/app/common/services/task-comment.service';
 import { taskService, alertService, taskComment, Task, commentsModal } from 'src/app/ajs-upgraded-providers';
-
-interface Comment {
-  recipient: {
-    name: string;
-  };
-  author_is_me: boolean;
-  recipient_read_time: string;
-  text: string;
-  last_read?: boolean;
-}
+import { TaskComment, TaskCommentService } from 'src/app/api/models/doubtfire-model';
+import { TaskCommentComposerData } from '../task-comment-composer/task-comment-composer.component';
 
 @Component({
   selector: 'task-comments-viewer',
   templateUrl: './task-comments-viewer.component.html',
   styleUrls: ['./task-comments-viewer.component.scss'],
 })
 export class TaskCommentsViewerComponent implements OnChanges, OnInit {
-  lastComment: Comment = { recipient: { name: '' }, author_is_me: false, recipient_read_time: '', text: '' };
+  // Get the comments body from the HTML template
+  @ViewChild('commentsBody') commentsBody: ElementRef;
+
+  lastComment: TaskComment;
   project: any = {};
   loading: boolean = true;
 
-  @Input() comment?: Comment = { recipient: { name: '' }, author_is_me: false, recipient_read_time: '', text: '' };
-  @Input() task: any = { comments: [] };
+  sharedComposerData: TaskCommentComposerData = {
```

Comment:
Thoughts on renaming this to be something like: `sharedCommentComposerData` just for clarity?